cmd/zoekt-mirror-github/main.go

```diff
@@ -247,6 +259,7 @@ func getOrgRepos(client *github.Client, org string, reposFilters reposFilters) (
 
 		opt.Page = resp.NextPage
 		repos = filterByTopic(repos, reposFilters.topics, reposFilters.excludeTopics)
+		repos = filterByState(repos, reposFilters.onlyActive)
```

Comment:
Instead of having a separate filter functions, can we combine them so that we only loop over all repos once?